src/com/facebook/buck/io/ExecutableFinder.java

```diff
@@ -169,6 +169,7 @@ private static boolean isExecutable(Path exe) {
 
     String pathEnv = env.get("PATH");
     if (pathEnv != null) {
+      pathEnv = pathEnv.trim();
```

Comment:
At appvoyer Path ends with the space (unfortunately). Then a java `Path` object is constructed. It is illegal to have trailing spaces in file names on windows.